.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
+Allowing for Kirby to be used in multiple projects without team Kirby being the bottleneck. 
+
+If you have a problem, an issue you want resolved, or even got brand new ideas for components. Well. There is no need to sit around and wait for team Kirby to get it implemented. 
+You can contribute yourself! And no matter your role, you can probably contribute with something. May it be a helpful comment or a bug report.
+
+We think that is neat. 
+
+There is still need for managing Kirby however. We need to make sure that the project is moving in the right direction. That submitted code is: high quality, not duplicating anything and consistent with the rest of the codebase.
+To do so a series of steps has been created that an issue must go through. All of these aim at making sure that the process of cooperating from idea to implementation is smooth for contributors. With as little time wasted as possible.
+
+<!-- General description of the contribution model -->
+In broad strokes the process is: submit an issue, get it refined, implement it, get it reviewed. Once reviewed it will automatically be merged to master. 
+
+Actionable checklists for these steps can be found further down in this section. The following highlight some of these steps to explain what they are and why they exist: 
+
+<!-- Why we do UX refinement?-->
+**UX Refinement**: If an issue involves visual changes then UX refinement must take place. This consists of UXers planning how the issue can be solved in a way that aligns with the design system. The `NOT UX refined`-label will be removed from the issue by team Kirby, once it has been done. If a pull-request with visual changes is submitted for an issue that has not been UX refined; there is a good chance that the UX review process may result in a bunch of changes. 
+
+<!--- Why do we do Tech refinement? -->
+**Tech Refinement**: This should happen before any code is written for an issue. Here it is discussed and agreed on how an issue should be implemented the best. How long it takes depends on the scope of the issue. For small issues it might be nothing more than an agreement via a direct message. Other times it is a full-blown meeting with multiple participants. The `NOT Tech refined`-label will be removed from the issue by team Kirby once it has been done. If a pull-request is submitted for an issue that has not been tech refined; changes and problems that could have been tackled from the beginning, might arise from the code-review. 
+
+**Publish a WIP draft**: We encourage contributors to publish a draft pull-request with a WIP implementation of their issue to get feedback. This can advantageously be done before tests are written. By doing this misunderstandings and issues can be tackled early on; here they are often easier to correct. If this is not done, they might not be discovered until the code-review. Tests might have to be changed. So therefore we encourage contributors to use the opportunity to get some feedback. 
```

Comment:
Suggestion: Add the word "early" or "timely" to the last sentence.

> So therefore we encourage contributors to use the opportunity to get some *timely* feedback.

And maybe somehow refer to the [agile principle of simplicity](https://agilemanifesto.org/principles.html) (my personal favourite)  😎

> Simplicity--the art of maximizing the amount of work not done--is essential.

Reply:
Timely is a nice word to throw in - It has been done 😄 
But can't seem to find a way to refer to the principle of simplicity, that does not feel like it is shoehorned in. The inbox is open for suggestions 👀

Reply:
You're right. It's not easy to merge into the text. Maybe we should consider visually large quotes. Something like:

![image](https://user-images.githubusercontent.com/150451/114991006-682e9400-9e99-11eb-8277-c14c2fc350d6.png)

Reply:
![image](https://user-images.githubusercontent.com/150451/114993804-73cf8a00-9e9c-11eb-8f61-0d7be38ea343.png)

can be done using `<blockquote>` and `<cite>` (but I think that is the same as a Markdown-to-HTML parser does with `>`).

The quote itself is maybe a more general point that's not only related to *Publish a WIP draft*. I'd love to throw it in somewhere, though 😁

Reply:
<img width="1025" alt="Screenshot 2021-04-16 at 11 05 21" src="https://user-images.githubusercontent.com/17269698/115001057-a466f200-9ea3-11eb-9f32-b8de6a69b2d1.png">

How is this 🤔?

Reply:
Great